### ISSUE TITLE
Migration issue from 0.2.16 to 0.2.18 (and 0.2.17)

### ISSUE THREAD

**Issue by Nexela** [Helfima/helmod#18]
helmod.lua:183 in on_configuration_changed
attempt to index field "model" a nil value

**Comment by Helfima** [comment]
ok i see the problem
i have no test on existing model

search
```
data.model.page = nil
data.model.step = nil
data.model.maxPage = nil
data.model.needPrepare = nil
data.model.products = nil
data.model.input = nil
data.model.recipes = nil
data.model.currentTab = nil
```
replace by
```
if data.model ~= nil then
	data.model.page = nil
	data.model.step = nil
	data.model.maxPage = nil
	data.model.needPrepare = nil
	data.model.products = nil
	data.model.input = nil
	data.model.recipes = nil
	data.model.currentTab = nil
end
```
i can't test I work but if you want try edit helmod.lua in the zip mod file (near: line 183)

**Comment by Helfima** [comment]
fixed on 0.2.19
